static/docs/get-started/add-files.md

```diff
@@ -16,47 +21,42 @@ into `data` subdirectory. To download, right-click
 
 </details>
 
-```dvc
-$ mkdir data
-$ wget https://dvc.org/s3/get-started/data.xml -O data/data.xml
-```
-
-To take a file (or a directory) under DVC control just run `dvc add`, it accepts
-any file or directory:
+To take a file (or a directory) under DVC control just run `dvc add` on it. For
+example:
 
 ```dvc
 $ dvc add data/data.xml
 ```
 
-DVC stores information about your data file in a special DVC-file, that has a
-human-readable [format](/doc/user-guide/dvc-file-format) and can be committed to
-Git to track versions of your file:
+DVC stores information about the added data in a special **DVC-file** that has a
+human-readable [format](/doc/user-guide/dvc-file-format). It can be committed to
+Git:
 
 ```dvc
 $ git add data/.gitignore data/data.xml.dvc
 $ git commit -m "add raw data to DVC"
 ```
 
+> Committing these special files to Git allows us to tack different versions of
```

Comment:
I would not make it a note. It's quite important piece of information.

Reply:
You're right, made into regular paragraph in 328b2ec